content/applications/sales/crm/performance/win_loss.rst

```diff
@@ -1,26 +1,249 @@
+wins and losses by salesperson or sales team to see who has the best conversion rate, whereas a
+marketing team might group by sources or medium to determine where their advertising has been most
+successful.
 
-.. image:: win_loss/win_loss01.png
-    :align: center
+.. seealso::
+   - :doc:`../acquire_leads/convert`
+   - :doc:`../acquire_leads/send_quotes`
+   - :doc:`../pipeline/lost_opportunities`
 
-You can also change the *Measures* to *Total Revenue*.
+Create a basic win/loss report
+==============================
 
-.. image:: win_loss/win_loss02.png
-    :align: center
+A basic win/loss report filters **the leads/opportunities from the past year, whether won or
+lost, and groups the results by their stage in the CRM pipeline**. Displaying this in the
+*Pipeline Analysis* requires a custom filter and the :guilabel:`Stage` grouping to create the
+following search criteria:
 
-You also have the ability to switch to a pie chart view.
+.. image:: win_loss/search-criteria-for-basic-win-loss.png
+   :align: center
+   :alt: The search criteria for win/loss reports is Created On, Stage, and Active is in true false.
 
-.. image:: win_loss/win_loss03.png
-    :align: center
+Follow the steps below to create a basic win/loss report using the above criteria:
+
+1. Navigate to :menuselection:`CRM app --> Reporting --> Pipeline`. At the *Pipeline Analysis* page,
+   click on the :guilabel:`Search bar` to open a menu of filters and grouping options.
+
+.. image:: win_loss/filters-for-basic-win-loss-report.png
+   :align: center
+   :alt: The Search menu containing the filters for a basic win/loss report.
+
+2. In the *Search* menu, under the *Group By* heading, click :guilabel:`Stage`. Under the *Filters*
+   heading, click :guilabel:`Add Custom Filter`.
+3. In the resulting pop-up menu, click the current filter (by default, it is set to
+   :guilabel:`Country`) and select :guilabel:`Active` from the drop-down. The rule should now read
+   "`Active` `is` `set`".
+4. Click :guilabel:`New Rule`, and once more change the default filter to :guilabel:`Active`. Click
+   on the last part of this rule and change it to read "`Active` `is` `not set`."
+5. Click :guilabel:`Add`.
+
+.. image:: win_loss/add-custom-active-filter.png
+   :align: center
+   :alt: The Add Custom Filter menu showing two rules: (1) Active is set, and (2) Active is not set.
+
+Click off the *Search* menu to reveal the report, which now displays the total :guilabel:`Count` of
+leads, whether won or lost, grouped by their stage in the CRM pipeline. Hover over a section of the
+report to see the number of leads in that stage.
+
+.. image:: win_loss/basic-win-loss-report.png
+   :align: center
+   :alt: A basic win/loss report showing all leads whether won or lost grouped by stage.
+
+Advanced filters and groupings
+------------------------------
+
+The *Pipeline Analysis* can be customized with various filters and group by options. After
```

Comment:
At this point, the reader is expected to be on the Pipeline Analysis page, so the use of GUI here is required -- because that title is actually on the screen